### DISCUSSION TITLE
Where is internal alerts metrics

### DISCUSSION THREAD

**Discussion by trungquan710** [SUSE/sap_host_exporter#72] [Q&A]
SUSE document shows that SAP Host exporter exports SAP internal alerts. But when I clone from github, install and run, I don't see any internal alert metrics.

 I read go files in  https://github.com/SUSE/sap_host_exporter)/collector/ and didn't find GetAlerts command

**Comment by stefanotorresi** [comment]
We don't export anything about the alerts. All the metrics are documented here: https://github.com/SUSE/sap_host_exporter/blob/main/doc/metrics.md

We could probably add the current number of alerts per severity easily, but I have a feeling this is not what you intended.

Could you please tell us which is this documentation page you're talking about?

**Reply by trungquan710** [reply]
I read about SAP host exporter in this page:
[https://documentation.suse.com/sles-sap/15-SP2/html/SLES-SAP-sol-monitoring/art-sol-monitoring.html](https://documentation.suse.com/sles-sap/15-SP2/html/SLES-SAP-sol-monitoring/art-sol-monitoring.html)

Maybe there's something mismatch here :D

**Reply by stefanotorresi** [reply]
Thanks for reporting this, we will update the official docs page by removing that mention of "internal alerts".
